src/transformers/pipelines/__init__.py

```diff
@@ -256,7 +257,7 @@
     "image-segmentation": {
         "impl": ImageSegmentationPipeline,
         "tf": (),
-        "pt": (AutoModelForImageSegmentation,) if is_torch_available() else (),
+        "pt": (AutoModelForImageSegmentation, AutoModelForSemanticSegmentation) if is_torch_available() else (),
```

Comment:
```suggestion
        "pt": (AutoModelForImageSegmentation, AutoModelForSemanticSegmentation) if is_torch_available() else (),
```
The `AutoModelForImageSegmentation` class is likely to be renamed to `AutoModelForPanopticSegmentation`, cc @FrancescoSaverioZuppichini .